include/deal.II/base/numbers.h

```diff
@@ -133,19 +133,8 @@ DEAL_II_NAMESPACE_CLOSE
 // Declare / Import auto-differentiable math functions in(to) standard
 // namespace before numbers::NumberTraits is defined
 #ifdef DEAL_II_WITH_ADOLC
-#  include <deal.II/differentiation/ad/adolc_math.h>
-
 #  include <adolc/adouble.h> // Taped double
```

Comment:
Ideally, I'd like to remove this too, but that's going to take some more work since we need `adouble`'s definition in some of the equality assertion checks.